sdk/edgeorder/Azure.ResourceManager.EdgeOrder/src/Generated/Models/ProductFamily.cs

```diff
@@ -48,51 +48,21 @@ public partial class ProductFamily
         /// <summary> Initializes a new instance of <see cref="ProductFamily"/>. </summary>
         internal ProductFamily()
         {
-            ImageInformation = new ChangeTrackingList<EdgeOrderProductImageInformation>();
-            FilterableProperties = new ChangeTrackingList<FilterableProperty>();
             ProductLines = new ChangeTrackingList<ProductLine>();
             ResourceProviderDetails = new ChangeTrackingList<ResourceProviderDetails>();
         }
 
         /// <summary> Initializes a new instance of <see cref="ProductFamily"/>. </summary>
-        /// <param name="displayName"> Display Name for the product system. </param>
-        /// <param name="description"> Description related to the product system. </param>
-        /// <param name="imageInformation"> Image information for the product system. </param>
-        /// <param name="costInformation"> Cost information for the product system. </param>
-        /// <param name="availabilityInformation"> Availability information of the product system. </param>
-        /// <param name="hierarchyInformation"> Hierarchy information of a product. </param>
-        /// <param name="filterableProperties"> list of filters supported for a product. </param>
         /// <param name="productLines"> List of product lines supported in the product family. </param>
         /// <param name="resourceProviderDetails"> Contains details related to resource provider. </param>
         /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
-        internal ProductFamily(string displayName, ProductDescription description, IReadOnlyList<EdgeOrderProductImageInformation> imageInformation, EdgeOrderProductCostInformation costInformation, ProductAvailabilityInformation availabilityInformation, HierarchyInformation hierarchyInformation, IReadOnlyList<FilterableProperty> filterableProperties, IReadOnlyList<ProductLine> productLines, IReadOnlyList<ResourceProviderDetails> resourceProviderDetails, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        internal ProductFamily(IReadOnlyList<ProductLine> productLines, IReadOnlyList<ResourceProviderDetails> resourceProviderDetails, IDictionary<string, BinaryData> serializedAdditionalRawData)
         {
-            DisplayName = displayName;
-            Description = description;
-            ImageInformation = imageInformation;
-            CostInformation = costInformation;
-            AvailabilityInformation = availabilityInformation;
-            HierarchyInformation = hierarchyInformation;
-            FilterableProperties = filterableProperties;
             ProductLines = productLines;
             ResourceProviderDetails = resourceProviderDetails;
             _serializedAdditionalRawData = serializedAdditionalRawData;
         }
 
-        /// <summary> Display Name for the product system. </summary>
```

Comment:
Missing a lot of properties, which from parents in the spec.
TSP: https://github.com/Azure/azure-rest-api-specs/blob/c15bc856fc0ec92b150a0c4f80bb9c022046d971/specification/edgeorder/EdgeOrder.Management/models.tsp#L1617

Reply:
Fixed